db/migrations/20230403094334-ss58-prefix-second-priority-data.js

```diff
@@ -0,0 +1,169 @@
+};
+
+const migrateExtrinsicTableArgs = async (db, ss58Format) => {
+    console.log('Start migration for args property for extrinsic table');
+
+    const result = await executeDbRunSqlAsPromise(db, `SELECT MIN(block_number), MAX(block_number) FROM extrinsic WHERE is_signed = TRUE;`);
+    const {min, max} = result.rows[0];
+
+    await migrateTableData(db, ss58Format, min, max, 'extrinsic', 'args');
+
+    console.log('✅ Finished migration for args property for extrinsic table');
+};
+
+const migrateTableData = async (db, ss58Format, min, max, table, property) => {
+    let startBlock = +min;
+    const endBlock = +max;
+    const batchSize = 50000;
+
+    while (startBlock < endBlock) {
+        const sql = `SELECT block_number, ${table}_index, ${property} FROM ${table} WHERE ${property} LIKE '%"%5%"%' AND block_number BETWEEN ${startBlock} AND ${startBlock + batchSize};`;
+        const {rows, rowCount} = await executeDbRunSqlAsPromise(db, sql);
+
+        startBlock += batchSize;
+
+        if (!rowCount) {
+            console.log(`Skipped for batch ${startBlock} -> ${endBlock + batchSize}`);
+        } else {
+            console.log(`Extracted ${rowCount} rows. Batch ${startBlock} -> ${startBlock + batchSize}`);
+
+            for (const row of rows) {
+                const { [property]: value, block_number, [table + '_index']: index } = row;
+                const nextValue = changeAccountInJSON(value, ss58Format);
+                if (value !== nextValue) {
+                    await executeDbRunSqlAsPromise(db, `UPDATE ${table} SET ${property}='${nextValue}' WHERE block_number=${block_number} AND ${table}_index=${index};`);
+                }
+            }
+            console.log(`Finished batching for ${startBlock} -> ${endBlock + batchSize}`);
+        }
+    }
+};
+
+const migrateExtrinsicTableSigners = async (db, ss58Format) => {
+    console.log('Start migration for signer property for extrinsic table');
+
+    const result = await executeDbRunSqlAsPromise(db, `SELECT DISTINCT signer FROM extrinsic WHERE signer LIKE '5%'`);
+
+    console.log(`Selected ${result.rowCount} signer from extrinsic table`);
+
+    for (const row of result.rows) {
+        const {signer} = row;
+        const nextAddress = decodeAddress(signer, ss58Format);
+
+        console.log(`Start migration for ${signer}`);
+
+        await executeDbRunSqlAsPromise(db, `UPDATE extrinsic SET signer='${nextAddress}' WHERE signer='${signer}'`);
+
+        console.log(`Finished migration for ${signer}`);
+    }
+
+    console.log('✅ Finished migration for signer property for exricnsic table');
+};
+
+const migrateBlockTable = async (db, ss58Format) => {
+    console.log('Start migration for Block table');
+
+    const result = await executeDbRunSqlAsPromise(db, `SELECT DISTINCT block_author FROM block WHERE block_author LIKE '5%'`);
+
+    console.log(`Selected ${result.rowCount} accounts from block table`);
+
+    await Promise.all(result.rows.map(async (row) => {
```

Comment:
Let's do it in series instead of parallelism to avoid spike load on DB.

Reply:
Done.
[Replace Promise.all with sequential await calls in migration code](https://github.com/Cerebellum-Network/cerestats/pull/130/commits/88a09665964b925408c6e5b0c45697f53f34d4bf)